package.json

```diff
@@ -38,8 +38,10 @@
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "sass-loader": "^11.0.1",
-    "style-loader": "^2.0.0",
     "webpack": "^5.35.1",
     "webpack-cli": "^4.6.0"
+  },
+  "devDependencies": {
+    "style-loader": "^2.0.0"
```

Comment:
This actually isnt a dev dep, but thats ok

Reply:
I think that one got added automatically, unless I installed funny.